uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml

```diff
@@ -216,6 +213,20 @@ Control {
                 }
             }
         }
+
+        Button {
+            anchors {
```

Comment:
:thought_balloon: `GridLayout` exists to both size and poition child content. I see you've removed it from the Layout. I think in this case, this is fine as you only need to position the close button and not size it.